boards/arm/stm32_min_dev/stm32_min_dev.yaml

```diff
@@ -9,3 +9,4 @@ ram: 20
 supported:
   - i2c
   - pwm
+  - usb_device
```

Comment:
Unfortunately, you can't actually configure this since this will trigger sample/subsys/usb/hid-mouse application build,
and this sample requires a button on the board.
Alternate (and cleanest) way to solve this issue would be to update sample/subsys/usb/hid-mouse yaml file in order not to build when button is not supported (it actually mentions button in "harness" field, but I doesn't seem to proceed to any filtering on that field).

Reply:
Probably it is not a good idea to mess up with samples within this thread.
Should I just remove this line?